tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -3,6 +3,9 @@ set -e
 
 SYS_LONG_BIT=$(getconf LONG_BIT)
 
+# Get the directory path of this script.
+CMD_HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
The above command string is actually the _simplest_ recommended way to get the script path for any cases that do not have a symlink as the last step! 

There are a few even more complex ways if we want to cover every edge-case and corner case, but those don't really apply inside this repo.

https://stackoverflow.com/a/246128/282326